source/jormungandr/jormungandr/tests/instance_manager_tests.py

```diff
@@ -77,3 +81,20 @@ def get_instances_by_object_id_test(manager, mocker):
         assert len(instances) == 1
         assert 'pdl' == instances[0].name
         assert mock.called
+
+
+def choose_best_instance_test():
+    """
+    Test to choose the best instance according to comparator : priority > is_free=False > is_free=True
+    """
+    instances_list = [
+        FakeInstance('fr-nw', is_free=True, priority=0),
+        FakeInstance('fr-nw-c', is_free=True, priority=0),
+    ]
+    assert choose_best_instance(instances_list).name == 'fr-nw'
```

Comment:
Not sure: here you test that you keep the same order as the list? Why is it useful to test it?

Reply:
Because if you look at the function 'choose_best_instance', that's what it does and it does happen in other tests. There's now a comment in the function description about it. 
We can maybe use the idea of alphabetical order here too.